src/platform/sdl/sdl_platform.cpp

```
@@ -9,6 +9,12 @@
 
 namespace moth_graphics::platform::sdl {
 
+    Platform::Platform() = default;
```

Clang (14.0.6)

[error] constructor does not initialize these fields: m_context

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)